mcx/jax.py

```diff
@@ -71,3 +74,58 @@ def unravel_list(arr):
     )
 
     return flat, unravel_list
+
+
+def define_tqdm(num_samples):
```

Comment:
I am not a fan of global state. Do you think there's a way to avoid this using e.g. a closure? If you don't let me know and I'll have a closer look.

Reply:
Good idea; just trying it out now